backend/app/views/spree/admin/reimbursements/show.html.erb

```diff
@@ -54,12 +54,12 @@
 </fieldset>
 
 <fieldset class="no-border-bottom">
-  <legend align='center'><%= Spree.t(:refunds) %></legend>
+  <legend align='center'><%= Spree::Refund.model_name.human %></legend>
   <table class="index reimbursement-refunds">
     <thead data-hook="customer_return_header">
       <tr>
         <th><%= Spree.t(:description) %></th>
```

Comment:
This can also be done using `human_attribute_name`, even though it isn't a database column. It's also kinda special, because it isn't really a description but the payment method's name, so I think it merits having its own translation.